x-pack/plugins/maps/public/reducers/ui.ts

```diff
@@ -15,19 +16,30 @@ import {
   SHOW_TOC_DETAILS,
   HIDE_TOC_DETAILS,
   UPDATE_INDEXING_STAGE,
+  // @ts-ignore
 } from '../actions/ui_actions';
 
-export const FLYOUT_STATE = {
-  NONE: 'NONE',
-  LAYER_PANEL: 'LAYER_PANEL',
-  ADD_LAYER_WIZARD: 'ADD_LAYER_WIZARD',
-};
+export enum FLYOUT_STATE {
+  NONE = 'NONE',
+  LAYER_PANEL = 'LAYER_PANEL',
+  ADD_LAYER_WIZARD = 'ADD_LAYER_WIZARD',
+}
+
+export enum INDEXING_STAGE {
+  READY = 'READY',
+  TRIGGERED = 'TRIGGERED',
+  SUCCESS = 'SUCCESS',
+  ERROR = 'ERROR',
+}
 
-export const INDEXING_STAGE = {
-  READY: 'READY',
-  TRIGGERED: 'TRIGGERED',
-  SUCCESS: 'SUCCESS',
-  ERROR: 'ERROR',
+export type UiState = {
```

Comment:
fwiw, possible to rename to `MapsUiState` ? The reason is that `UiState` is everywhere in the rest of Kibana (UpTime, visualizations, ...).